scene/main/node.h

```diff
@@ -327,6 +327,11 @@ class Node : public Object {
 	GDVIRTUAL1(_unhandled_input, Ref<InputEvent>)
 	GDVIRTUAL1(_unhandled_key_input, Ref<InputEvent>)
 
+#ifndef DISABLE_DEPRECATED
+	void set_name_76560(const String &p_name);
```

Comment:
IIRC the `_bind_compatibility_methods` generally is listed first

Reply:
In the current code, it's listed after in every case.